browser/css/btns.css

```diff
@@ -50,14 +65,16 @@
 	background-color: var(--color-background-darker);
 	border-color: var(--color-border-darker);
 }
+
 /* toolbuttons selected hover */
+.hasnotebookbar .ui-content.unotoolbutton.has-label:hover,
 .hasnotebookbar .ui-content.unotoolbutton.selected:hover,
 .unotoolbutton.notebookbar:hover,
 .unotoolbutton.jsdialog:hover,
 .hasnotebookbar .ui-content.unotoolbutton.selected:not(.has-label):not(.inline):hover,
 .sidebar.unotoolbutton.selected:hover,
+.notebookbar.unotoolbutton.selected:hover,
 .jsdialog.unotoolbutton.selected:hover {
-	background-color: var(--color-background-darker) !important;
```

Comment:
We move the responsability of the `background-color` to the children since a button could have 2 parts